test/integration/acceptance/acceptance_fixture.cpp

```diff
@@ -5,11 +5,20 @@
 
 #include "integration/acceptance/acceptance_fixture.hpp"
 
+#include <utility>
+
 #include "datetime/time.hpp"
-#include "framework/integration_framework/integration_test_framework.hpp"
 #include "framework/specified_visitor.hpp"
 #include "utils/query_error_response_visitor.hpp"
 
+namespace {
+  template <typename T>
+  void checkStatus(const shared_model::interface::TransactionResponse &resp) {
+    ASSERT_NO_THROW(
```

Comment:
Assertions are better done in the test body, so that log contains a reference to line of test code.